.github/workflows/CI.yml

```diff
@@ -432,7 +433,7 @@ jobs:
         id: versioning
         run: |
           # build a version with github run id so that we can distingush each build for integ-test
-          Version="`cat VERSION`-$GITHUB_RUN_ID"
+          Version="`cat VERSION`-$GITHUB_SHA"
```

Comment:
Would rather not have the whole SHA. They can get rather long.
```suggestion
          Version="$(cat VERSION)-$(git rev-parse --short HEAD)"
```

Reply:
Interesting. Nice catch for getting the short SHA though.